haddock-api/src/Haddock/Backends/LaTeX.hs

```diff
@@ -444,11 +444,11 @@ ppLPatSig doc docnames ty unicode
 -- arguments as needed.
 ppTypeOrFunSig :: HsType DocNameI
                -> DocForDecl DocName  -- ^ documentation
-               -> ( LaTeX             -- ^ first-line (no-argument docs only)
-                  , LaTeX             -- ^ first-line (argument docs only)
-                  , LaTeX             -- ^ type prefix (argument docs only)
+               -> ( LaTeX             --   first-line (no-argument docs only)
+                  , LaTeX             --   first-line (argument docs only)
+                  , LaTeX             --   type prefix (argument docs only)
                   )
-               -> Bool                -- ^ unicode
+               -> Bool                --   unicode
```

Comment:
That bit of haddocks shouldn't be problematic no?

Reply:
Thanks for noticing! Phew.